test/e2e/toolchaincluster_test.go

```diff
@@ -174,3 +176,86 @@ func caBundle(bundle string) clusterOption {
 		c.Spec.CABundle = bundle
 	}
 }
+
+func TestForceSynchronization(t *testing.T) {
```

Comment:
```suggestion
func TestForceMetricsSynchronization(t *testing.T) {
```
Maybe we can change this func name to `TestForceMetricsSynchronization`? I'm not sure it would be immediately obvious that this test is about metrics since it's in toolchaincluster_test.go